.gitignore

```diff
@@ -8,6 +8,7 @@ Source/Core/Common/scmrev.h
 /[Bb]uild*/
 /[Bb]inary/
 /obj/
+/Source/Android/app/.cxx/
```

Comment:
This should probably have an accompanying comment to provide context for the listing in the gitignore. Looks fine otherwise.

Reply:
I don't really much other than that it gets created than building for Android, though... And it does say Android in the path already. Do you have a suggestion for what to write in the comment?

Reply:
A suitable comment would just mention that Android Studio creates files in that directory. Similar to the comment under it indicating that we ignore certain extensions because they're ignorable files creates by Visual Studio.

Reply:
Does the current commit look good?

Reply:
Yup! 👌